recipes/dnet/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.16.4":
+    url: "https://github.com/Jihadist/libdnet/archive/19c7040bc92ddf04139a6b566d811d8cd9c3d736.tar.gz"
```

Comment:
@memsharded thank you for notice. As you can see here, I used my own fork with some changes. These changes were merged in upstream repo https://github.com/ofalk/libdnet but not yet released.

Reply:
Ok, I didn't realize that this was using a fork origin.
Recipes in ``conan-center-index`` must use the original library upstream repo, not user forks, except in case of severe abandoned repos that are being actively maintained by someone else. In this case, it would be mandatory to use https://github.com/ofalk/libdnet origin.  Please change the sources origin. If some patch is absolutely necessarily to package it, it can be provided separately as a patch.